README.md

```diff
@@ -50,6 +50,29 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
 * (optional) to enable fedora internal audit capability, edit `install_scripts/config` and change the FEDORA_AUDIT variable to true. The FEDORA_AUDIT_LOCATION can also be changed from its default "/audit", if necessary.
 * `vagrant up`
 
+#### Using the backup and restore scripts
+The scripts at the ~/backup_restore directory can be used to test backing up and restoring the fedora repository for consistency.
+
+The following command will cause 50 parallel processes to load data to the repository while creating snapshots of fcrepo home directory every 2 seconds.
+
+```
+cd ~/backup_restore/
+./hot_backup_runner.sh 50 /var/lib/tomcat7/fcrepo4-data 2
```

Comment:
@awoods, the amount of space required depends on the input params. For example, ```./hot_backup_runner.sh 50 /var/lib/tomcat7/fcrepo4-data 2``` will run ```50``` parallel ingest processes and create backups every ```2``` seconds. You can reduce the backup frequency to 5 or 10 seconds, if you are running out of storage.